src/player.cpp

```diff
@@ -2188,6 +2188,11 @@ bool Player::removeVIP(uint32_t vipGuid)
 
 bool Player::addVIP(uint32_t vipGuid, const std::string& vipName, VipStatus_t status)
 {
+	if (getGUID() == vipGuid) {
```

Comment:
I think you can just use `guid` since this variable is available from `this`
sorry for bothering you so much 🤗

Someone else can come and say that something else is better so don't scold me later ;)

Reply:
> I think you can just use `guid` since this variable is available from `this`

That makes sense xD. I'll make the changes and commit in a moment.

> sorry for bothering you so much

Not bothering at all, I appreciate the feedback from experienced developers.